ta/supp_plugin/include/ta_supp_plugin_test.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021 Open Mobile Platform LLC
+ */
+
+#ifndef TA_SUPP_PLUGIN_TEST_H
+#define TA_SUPP_PLUGIN_TEST_H
+
+#define TA_SUPP_PLUGIN_TEST_UUID { 0x380231ac, 0xfb99, 0x47ad, \
```

Comment:
Please rename to `TA_SUPP_PLUGIN_UUID`, the `TEST` part is redundant.
Please drop the `_test` part from the name of this file too.